From 5227f8e5efb01ce70026b0dace77f4469e4f720d Mon Sep 17 00:00:00 2001
From: Greg Rose <gvrose@VMware-box.domain>
Date: Tue, 24 Mar 2020 08:42:02 -0700
Subject: [PATCH 09/19] compat: Fix nf_ip_hook parameters for RHEL 8

A RHEL release version check was only checking for RHEL releases
greater than 7.0 so that ended up including a compat fixup that
is not needed for 8.0.  Fix up the version check.

Signed-off-by: Greg Rose <gvrose8192@gmail.com>
Acked-by: Yi-Hung Wei <yihung.wei@gmail.com>
Signed-off-by: William Tu <u9012063@gmail.com>
---
 datapath/linux/compat/stt.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/datapath/linux/compat/stt.c b/datapath/linux/compat/stt.c
index 7b46d1a20..8a5853f19 100644
--- a/datapath/linux/compat/stt.c
+++ b/datapath/linux/compat/stt.c
@@ -1559,7 +1559,7 @@ static void clean_percpu(struct work_struct *work)
 #endif
 
 #ifdef HAVE_NF_HOOK_STATE
-#if RHEL_RELEASE_CODE > RHEL_RELEASE_VERSION(7,0)
+#if RHEL_RELEASE_CODE > RHEL_RELEASE_VERSION(7,0) && RHEL_RELEASE_CODE < RHEL_RELEASE_VERSION(8,0)
 /* RHEL nfhook hacks. */
 #ifndef __GENKSYMS__
 #define LAST_PARAM const struct net_device *in, const struct net_device *out, \
-- 
2.17.1

